packages/grafana-ui/src/components/index.ts

```diff
@@ -102,7 +102,9 @@ export { ClickOutsideWrapper } from './ClickOutsideWrapper/ClickOutsideWrapper';
 export * from './SingleStatShared/index';
 export { CallToActionCard } from './CallToActionCard/CallToActionCard';
 export { ContextMenu, ContextMenuProps } from './ContextMenu/ContextMenu';
-export { Menu, MenuItem, MenuItemsGroup } from './Menu/Menu';
+export { Menu, MenuProps } from './Menu/Menu';
+export { MenuGroup, MenuGroupProps, MenuItemGroup } from './Menu/MenuGroup';
+export { MenuItem, MenuItemProps } from './Menu/MenuItem';
```

Comment:
I think you don't need to export the *Props interfaces as long as these are not used in the core app.